components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -106,12 +106,12 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
   return (
     <form onSubmit={handlePostHighlight} className="flex flex-col flex-1 gap-4">
       <Collapsible onOpenChange={setIsDivFocused} open={isDivFocused}>
-        <div className="flex flex-col gap-2 p-2 overflow-hidden text-sm bg-white border rounded-lg ">
+        <div className="flex flex-col gap-2 p-2 overflow-hidden text-sm bg-white border rounded-lg">
           <CollapsibleTrigger asChild>
             <div className="flex pr-2">
               <input
                 value={title}
-                maxLength={50}
+                maxLength={300}
```

Comment:
Please returns this to 50. We need to considere more aspects of the UX before making a changing like this.

Reply:
@bdougie done.